polar2grid/glue.py

```diff
@@ -171,9 +174,20 @@ def write_scene(scn, writers, writer_args, datasets, to_save=None):
         # no datasets to save
         return to_save
 
+    for dataID in datasets:
+        try:
+            area_def = getattr(scn[dataID], 'area', None)
+        except KeyError:
+            msg = 'No area definition in {}'.format(dataID)
+            LOG.info(msg)
+            LOG.error('Information', exc_info=True)
+            
+        if isinstance(area_def, SwathDefinition): 
+            scn[dataID].attrs['area'].area_id = "native"
+
     for writer_name in writers:
         wargs = writer_args[writer_name]
-
+        print(writer_name, datasets, wargs)
```

Comment:
Extra print

Reply:
Found it!  I thought I had gotten rid of all of those...